src/js/views/admin/program/flow/flow_views.js

```diff
@@ -0,0 +1,111 @@
+import Radio from 'backbone.radio';
+import hbs from 'handlebars-inline-precompile';
+import { View } from 'marionette';
+
+import { OwnerComponent, PublishedComponent } from 'js/views/admin/actions/actions_views';
```

Comment:
@paulfalgout, is it ok to pull these from the actions views, or should we duplicate them in here and make some tweaks so that they're program flow-specific (things like classnames)?

Reply:
yeah I think this is good.  might need some reorg down the line.  I'm seeing similar issues on the flow sidebar.